pom.xml

```diff
@@ -0,0 +1,385 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.mesosphere.velocity</groupId>
+	<artifactId>velocity</artifactId>
+	<version>1.2-SNAPSHOT</version>
```

Comment:
Assuming that this is going to be the 1.0.0 release, we should update the version here